WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1695483254.305787 2400215 tfrt_cpu_pjrt_client.cc:349] TfrtCpuClient created.
No GPU/TPU found, falling back to CPU. (Set TF_CPP_MIN_LOG_LEVEL=0 and rerun for more info.)
/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/scvi/_settings.py:63: UserWarning: Since v1.0.0, scvi-tools no longer uses a random seed by default. Run `scvi.settings.seed = 0` to reproduce results from previous versions.
  self.seed = seed
/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/scvi/_settings.py:70: UserWarning: Setting `dl_pin_memory_gpu_training` is deprecated in v1.0 and will be removed in v1.1. Please pass in `pin_memory` to the data loaders instead.
  self.dl_pin_memory_gpu_training = (
Traceback (most recent call last):
  File "/home/icb/alessandro.palma/environment/scCFM/scCFM/train_hydra/utils/exceptions.py", line 11, in __call__
    return self.f(*args, **kwargs)
  File "/home/icb/alessandro.palma/environment/scCFM/scCFM/train_hydra/train_cfm.py", line 43, in main
    datamodule = TrajectoryDataModule(path=config.datamodule.path,
  File "/home/icb/alessandro.palma/environment/scCFM/scCFM/datamodules/time_sc_datamodule.py", line 110, in __init__
    self.dataset = TrajectoryDataset(path,
  File "/home/icb/alessandro.palma/environment/scCFM/scCFM/datamodules/time_sc_datamodule.py", line 22, in __init__
    assert os.path.exists(path), "The data path does not exist"
AssertionError: The data path does not exist
